### PR TITLE
Focus Provider Registration Fix

### PR DESCRIPTION
**Pull request by StephenHodgson**
Overview
---
Fixed an issue where Focus Provider was not registering pointers for input sources that were already detected by the time our Focus Provider is started.

### CODE REVIEW

File: Assets/MixedRealityToolkit-SDK/Features/Input/FocusProvider.cs [hunk 1]
```diff
@@ -432,6 +437,23 @@ public bool RegisterPointer(IMixedRealityPointer pointer)
             return true;
         }
 
+        private void RegisterPointers(IMixedRealityInputSource inputSource)
```
Comment (SimonDarksideJ) [inline Assets/MixedRealityToolkit-SDK/Features/Input/FocusProvider.cs]: Can we make these RegisterPointer(s) calls more generic and put them in the InputSystem?
Seeing this duplicated / replicated around the project.

Comment (StephenHodgson) [reply Assets/MixedRealityToolkit-SDK/Features/Input/FocusProvider.cs]: That's a much larger breaking change than a simple bug fix.

Comment (SimonDarksideJ) [reply Assets/MixedRealityToolkit-SDK/Features/Input/FocusProvider.cs]: Fair enough, make a task for it later :D

Comment (StephenHodgson) [reply Assets/MixedRealityToolkit-SDK/Features/Input/FocusProvider.cs]: Much appreciated. I do agree we can refine this class a bit more and possibly make it a full proper manager. For now I want to get the rest of the input devices working first.

**Pull request merged**